docs/definitionOfdone.md

```diff
@@ -0,0 +1,32 @@
+# Definition of done
+
+The scope of this document is to clearly communicate the Auro team's definition of done.
+
+> Definition of Done: is a formal description of the state of the Increment when it meets the quality measures required for the product. The moment a Product Backlog item meets the Definition of Done, an Increment is born. The Definition of Done creates transparency by providing everyone a shared understanding of what work was completed as part of the Increment. If a Product Backlog item does not meet the Definition of Done, it cannot be released or even presented at the Sprint Review.
+
+[-- Scrum.org: Scrum Glossary](https://www.scrum.org/resources/scrum-glossary)
+
+
+## Team definition of done
+
+The following items are to be considered the Auro team's agreed upon definition of done with optional exceptions based on scope of story.
+
+1. **Code is complete:** The work submitted by the author is to be considered complete and there are no remaining items to be added to the scope of this update.
```

Comment:
Should we include in here the need for an issue prior to working on and submitting a PR? And should that be a pre-commit question?